src/components/pages/ProfilePage/components/ArrowBack/ArrowBack.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+import { ArrowBackProps } from '../../types';
+import s from './ArrowBack.module.css';
+
+export const ArrowBack: FC<ArrowBackProps> = ({ onClick }) => {
```

Comment:
You can import svg as ReactComponent and update css to update styles.

```suggestion
import { ArrowBackProps } from '../../types';
import { ReactComponent as Arrow } from './ArrowBack.svg';

import s from './ArrowBack.module.css';

export const ArrowBack = ({ onClick }: ArrowBackProps) => {
  return <Arrow className={s.arrowBack} onClick={onClick} />;
};
```

Reply:
подумлось, что по факту, этот компонент вообще не нужен. просто импортировать Arrow где надо...

Reply:
ну да))